src/util/mod.rs

```diff
@@ -30,18 +32,23 @@ impl DataWriter {
 
     /// Creates a new file in the writers directory with each point written
     /// on a separate line.
-    pub fn write(&mut self, points: Vec<Point>) {
```

Comment:
Why are you changing the existing method? I thought you were going to define a new method to be able to save list of bodies?

Reply:
I changed it from points: Vec<Point> to bodies: Vec<Body>

Reply:
Yes I see that, but why did you change it?

Reply:
I thought if the concept for bodies is similar to points, so I changed it ^^! so I cant use your already defined function ?

Reply:
But we still need the method to write only the points. If you change it, we no longer have this method anymore.